index.html

```diff
@@ -4,11 +4,21 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vanilla Todo</title>
-    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="css/normalize.css">
```

Comment:
```suggestion
    <link href="
https://cdn.jsdelivr.net/npm/normalize.css@8.0.1/normalize.min.css
" rel="stylesheet">
```
normalize.css 파일을 가져와서 잘 제공해주시는 것 같아요!
이 방식도 좋구 CDN을 활용한 link 태그로 적용해주는 방법도 알고 계시면 좋을 것 같습니다